### PR TITLE
Fixing docker build bug

### PR DESCRIPTION
**Pull request by bannsec**
Docker build has been broken for like a month. The .config directory doesn't exist at the point where it tries to chmod it, so i'm ensuring that it's present. The -p will also not cause the build to fail in the case that the directory does exist too.

### CODE REVIEW

**Comment by radare** [comment]
That patch must be in r2pm

**Comment by radare** [comment]
pls

**Comment by bannsec** [comment]
Think that should do it. Difficult to test since the Dockerfile git pulls instead of COPY from the submitted changes.

File: binr/r2pm/r2pm [hunk 1]
```diff
@@ -145,10 +145,10 @@ commitSuicide() {
 		exit 1
 	fi
 	countDown "Self destroying in" 3
-	confirm "> Delete ~/config.radare2" && (
+	confirm "> Delete ~/.config.radare2" && (
```
Comment (radare) [inline binr/r2pm/r2pm]: Why . And not /?

File: binr/r2pm/r2pm [hunk 2]
```diff
@@ -145,10 +145,10 @@ commitSuicide() {
...
 		rm -rf ~/.config/radare2
 	)
-	R2PATHS="${PREFIX} /usr /usr/local /opt/radare2 ~/config/radare2/prefix /"
+	R2PATHS="${PREFIX} /usr /usr/local /opt/radare2 ~/.config/radare2/prefix /"
```
Comment (radare) [inline binr/r2pm/r2pm]: Take this path from r2 -H instead of hardcoding it

File: binr/r2pm/r2pm [hunk 3]
```diff
@@ -145,10 +147,10 @@ commitSuicide() {
 		exit 1
 	fi
 	countDown "Self destroying in" 3
-	confirm "> Delete ~/.config.radare2" && (
-		rm -rf ~/.config/radare2
+	confirm "> Delete $RHOMEDIR" && (
+		rm -rf $RHOMEDIR
```
Comment (radare) [inline binr/r2pm/r2pm]: use quotes, otherwise it may be a problem if the path contain spaces

File: binr/r2pm/r2pm [hunk 4]
```diff
@@ -145,10 +147,10 @@ commitSuicide() {
...
 	)
-	R2PATHS="${PREFIX} /usr /usr/local /opt/radare2 ~/.config/radare2/prefix /"
+	R2PATHS="${PREFIX} /usr /usr/local /opt/radare2 ${RHOMEDIR}/prefix /"
```
Comment (radare) [inline binr/r2pm/r2pm]: uhm if this path contain spaces, we cant iterate over those paths like this :? and i dont want to use bashisms, so we may probably use : as token to split that list

Comment (bannsec) [reply binr/r2pm/r2pm]: Not sure how to do that.

File: binr/r2pm/r2pm [hunk 5]
```diff
@@ -162,6 +164,7 @@ case "$1" in
 init|up|update)
 	[ -z "$R2PM_DBDIR" ] && R2PM_DBDIR="${R2PM_USRDIR}/db"
 	mkdir -p "${R2PM_GITDIR}"
+	mkdir -p ${HOME}/.config
```
Comment (radare) [inline binr/r2pm/r2pm]: home can contain spaces, please use ""

File: binr/r2pm/r2pm [hunk 6]
```diff
@@ -145,10 +147,10 @@ commitSuicide() {
 		exit 1
 	fi
 	countDown "Self destroying in" 3
-	confirm "> Delete ~/config.radare2" && (
-		rm -rf ~/.config/radare2
+	confirm "> Delete $RHOMEDIR" && (
+		rm -rf $RHOMEDIR
```
Comment (radare) [inline binr/r2pm/r2pm]: RHOMEDIR can contain spaces, please use ""

File: binr/r2pm/r2pm [hunk 7]
```diff
@@ -145,10 +147,10 @@ commitSuicide() {
...
 	)
-	R2PATHS="${PREFIX} /usr /usr/local /opt/radare2 ~/config/radare2/prefix /"
+	R2PATHS="${PREFIX} /usr /usr/local /opt/radare2 ${RHOMEDIR}/prefix /"
```
Comment (radare) [inline binr/r2pm/r2pm]: use : as separator, $RHOMEDIR can contain spaces and will make this thing fail

Comment (radare) [reply binr/r2pm/r2pm]: IFS=:

**Comment by bannsec** [comment]
Changes are implemented.

**Comment by radare** [comment]
Thanks!

**Pull request merged**